modules/virt-supported-ssp-tasks.adoc

```diff
@@ -37,3 +40,7 @@ The following table shows the tasks that are included as part of the Tekton Task
 | `wait-for-vmi-status`
 | Wait for a specific status of a virtual machine instance and fail or succeed based on the status.
 |===
+[.small]
+--
+1. Template based tasks are planned to be deprecated.
```

Comment:
they are already depracated in 4.14. We now have tasks creating VM based on  clusterinstancetype and clusterpreference.

Reply:
sorry what i mean is pipeline are using tasks  based on clusterinstancetype and clusterpreference for creating VM. 
template based tasks still exist but not part of pipelines tasks (we don't use them with default three templates)

Reply:
@geetikakay if my understanding of the change is correct, how does this look as a note instead:

1. Virtual machine creation in pipelines now utilize `ClusterInstanceType` and `ClusterPreference` instead of template-based tasks which have been deprecated. Commands `create-vm-from-template`, `copy-template`, and `modify-vm-template` remain available but are not used in default pipeline tasks.

of course if my understanding is still wrong, feel free to correct me. thanks!

Reply:
this is correct